Framework/Algorithms/test/PolarizationCorrectionWildesTest.h

```diff
@@ -1467,28 +1425,24 @@ class PolarizationCorrectionWildesTest : public CxxTest::TestSuite {
     AnalysisDataService::Instance().addOrReplace(wsName, ws);
   }
 
-  void prepareAndRegisterWorkspaces(std::vector<std::string> &wsNames,
-                                    const std::vector<Mantid::API::MatrixWorkspace_sptr> &wsList, const size_t nHist) {
+  void setWorkspacesTestData(std::vector<std::string> &wsNames,
+                             const std::vector<Mantid::API::MatrixWorkspace_sptr> &wsList, const size_t nHist) {
 
     for (size_t i = 0; i != wsNames.size(); ++i) {
       for (size_t j = 0; j != nHist; ++j) {
         wsList[i]->mutableY(j) *= static_cast<double>(i + 1);
         wsList[i]->mutableE(j) *= static_cast<double>(i + 1);
```

Comment:
```suggestion
  void setWorkspacesTestData( const std::vector<Mantid::API::MatrixWorkspace_sptr> &wsList) {

    for (size_t i = 0; i < wsList.size(); ++i) {
      for (size_t j = 0; j < wsList[i]->getNumberHistograms(); ++j) {
        wsList[i]->mutableY(j) *= static_cast<double>(i + 1);
        wsList[i]->mutableE(j) *= static_cast<double>(i + 1);
```
I think you can get away with only passing the `wsList` here and in another prepare workspace function.